server/fly.io/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "cargo-quickinstall-stats-server"
+version = "0.0.0"
+authors = ["David Laban <alsuren@gmail.com>"]
+# based on https://github.com/fly-apps/hello-rust by Jerome Gravel-Niquet <jeromegn@gmail.com>
+edition = "2021"
+
+# Make a new workspace so that we get our own Cargo.lock and target dir for the docker build. 
+[workspace]
+
+[dependencies]
+tokio = { version = "1", features = ["full"] }
```

We don't need the `full` feature set:

```suggestion
tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
```

We could also remove `macros` dep if you adopt my other suggestion of constructing `tokio::runtime::Runtime` manually to spawn the future.